src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/incentives/IncentivesResource.kt

```diff
@@ -22,12 +27,15 @@ class IncentivesResource(private val telemetryClient: TelemetryClient, private v
   @OptIn(DelicateCoroutinesApi::class)
   @PutMapping("/incentives/reports/reconciliation")
   @ResponseStatus(HttpStatus.ACCEPTED)
-  fun generateIncentiveReconciliationReport() {
-    telemetryClient.trackEvent("incentives-reports-reconciliation-requested")
-    log.info("Incentives reconciliation report requested")
+  suspend fun generateIncentiveReconciliationReport() {
+    val activePrisonersCount = nomisApiService.getActivePrisoners(0, 1).totalElements
```

Comment:
decided to move the upfront count to before we spawn a corotiune - feels nicer and we get better telemetry